### ISSUE TITLE
Unit tests

### ISSUE THREAD

**Issue by jackpoz** [jackpoz/BotFarm#12]
Create unit tests for these issues:
- [ ] https://github.com/TrinityCore/TrinityCore/issues/15633

**Comment by dmpaul26** [comment]
Do I just write the tests in the tests folder of the test branch?

**Comment by jackpoz** [comment]
there's no tests folder or test branch, what are you talking about ? Currently there's https://github.com/jackpoz/BotFarm/tree/master/TrinityCore%20UnitTests project but it doesn't have any particular unit test.

**Comment by dmpaul26** [comment]
Oh wow. I followed this issue from https://github.com/TrinityCore/TrinityCore/pull/16279. I didn't realize this was in a different repo. That PR is waiting for testing I assumed, would they be written here or in the TC repo?

**Comment by jackpoz** [comment]
TC repo currently doesn't have any kind of unit testing

**Comment by dmpaul26** [comment]
Yeah, that's what I thought too, that's why I was lost. Thanks, sorry for this.
